s3_upload/utils/utils.py

```diff
@@ -420,8 +466,25 @@ def verify_config(config) -> None:
     if not isinstance(config.get("max_threads", 0), int):
         errors.append("max_threads must be an integer")
 
-    if not config.get("log_dir"):
```

Comment:
because it is optional and therefore would break if not set in the config, also when the code runs it first checks that the given log directory exists and is writable (defaulting to `/var/log/s3_upload` if not set in the config)
<!-- Reviewable comment -OCYoJNXESCg0ctvPJqO:b-w2bvy2 -->
<!-- Sent from Reviewable.io -->